tests/test_gitpuller.py

```diff
@@ -99,7 +99,11 @@ def test_initialize():
 def command_line_test_helper(remote_path, branch, pusher_path):
     work_dir = "/".join(os.path.dirname(os.path.abspath(__file__)).split("/")[:-1]) + "/nbgitpuller"
     try:
-        cmd = ['python3', 'pull.py', remote_path, branch, pusher_path]
+        cmd = ['python3', 'pull.py', remote_path]
+        if branch is not None:
+            cmd += ['--branch_name', branch]
```

Comment:
```suggestion
            cmd += ['--branch-name', branch]
```

Reply:
I pushed another commit into the PR:
- branch_name is optional but remains positional
- repo_dir becomes --target-dir -- I did not change repo_dir to target_dir in the rest of the code base; I could but repo_dir makes sense to the developer in the context but I agree could make less sense or no sense to a user invoking from the command-line.
Thanks for the feedback.